openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -509,9 +515,12 @@ export class ResponseView {
 
       const view = this;
       const title = gettext('Confirm Submit Response');
-      // Keep this on one big line to avoid gettext bug: http://stackoverflow.com/a/24579117
-      // eslint-disable-next-line max-len
-      const msg = gettext('You\'re about to submit your response for this assignment. After you submit this response, you can\'t change it or submit a new response.');
+      const msg = gettext(
+        'You\'re about to submit your response for this assignment. '
+        + 'After you submit this response, you can\'t change it or '
+        + 'submit a new response, unless the instructor has activated '
+        + 'the resubmission feature.',
+      );
```

Comment:
Should we let the student know about the resubmission feature if it's not on? The student doesn't know about resubmissions unless it's on, so I'm not sure we should inform them. What do you think?

Reply:
With this new implementation, the message displayed would not be entirely correct, because the student could resubmit if the conditions are met.

Reply:
We can at least tell whether resubmission has been enabled.. I think it would be clearer if we only mentioned resubmission if it's possible. e.g.

```
     if (resubmission_allowed) {
      const msg = gettext(
        'You\'re about to submit your response for this assignment. '
        + 'After you submit this response, you may have a limited time to resubmit before your submission is graded.'
       );
     }
     else {
          const msg = gettext('You\'re about to submit your response for this assignment. After you submit this response, you can\'t change it or submit a new response.');
     }
```